src/bundle/Resources/views/themes/admin/ui/global_search.html.twig

```diff
@@ -3,12 +3,19 @@
 {% trans_default_domain 'ibexa_search' %}
 
 <div class="ibexa-global-search">
+    {% set is_mac = 'Mac OS X' in app.request.headers.get('user-agent') %}
```

Comment:
Considering I do something similar here https://github.com/ibexa/page-builder/pull/284/files#diff-2b508d485ebbf809ce37877bfb5bd43eb08a79e12ee4ccf2b98f7bae6ff66438R136 maybe it would be good to make some helper/macro/twig func from it?